kernel/kernel-api/src/main/java/io/delta/kernel/internal/InternalScanFileUtils.java

```diff
@@ -87,6 +89,11 @@ private InternalScanFileUtils() {}
 
   private static final int ADD_FILE_DV_ORDINAL = ADD_FILE_SCHEMA.indexOf("deletionVector");
 
+  private static final int ADD_FILE_BASE_ROW_ID_ORDINAL = ADD_FILE_SCHEMA.indexOf("baseRowId");
+
+  private static final int ADD_FILE_DEFAULT_ROW_COMMIT_VERSION_ORDINAL =
```

Comment:
Let's delete these and use the AddFile class you created in your previous PR! This will reduce the coupling between these two classes.